frontend/src/utils/sort.ts

```diff
@@ -0,0 +1,33 @@
+import { Priority } from '../api/models'
+
+export type SortDirection = 'ascending' | 'descending' | 'none'
+
+export const sort = (a: string, b: string, sortDirection: SortDirection) => {
```

Comment:
```suggestion
export const sortString = (a: string, b: string, sortDirection: SortDirection) => {
```

Or include number, boolean, etc. as input types.

Reply:
good point, will put number and boolean as input types